warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__shapes_valid.sql

```diff
@@ -0,0 +1,49 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('stg_gtfs_guidelines__feed_guideline_index') }}
+    WHERE check = {{ shapes_valid() }}
+),
+
+-- For this check we are only looking for errors and warnings related to shapes
+validation_fact_daily_feed_codes_shape_related AS (
+    SELECT * FROM {{ ref('validation_fact_daily_feed_codes') }}
+    WHERE code IN (
```

Comment:
optional suggestion: This `where` clause could just be added to the next CTE to avoid having a whole separate CTE. Totally optional / style question